jac-uk/admin
# Bugfix: #2108 Table overflow

## What's included?
Add a scrollbar to the Table component if the table is overflowing.

Closes #2108 

Note: this work is related to [jac-kit: Bugfix: #2108 Table overflow](https://github.com/jac-uk/jac-kit/pull/139).

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Go to the following list and resize the browser window to a small size. Check if you can see all the content in the table.
- Events: https://jac-admin-develop--pr2109-bugfix-2108-table-ov-ds494l0q.web.app/events
- Notifications: https://jac-admin-develop--pr2109-bugfix-2108-table-ov-ds494l0q.web.app/notifications
- Exercises: https://jac-admin-develop--pr2109-bugfix-2108-table-ov-ds494l0q.web.app/exercises
- Candidates: https://jac-admin-develop--pr2109-bugfix-2108-table-ov-ds494l0q.web.app/candidates/
- Applications: https://jac-admin-develop--pr2109-bugfix-2108-table-ov-ds494l0q.web.app/exercise/ONqUjAzhDS7rjh6dFJLB/applications/applied
- Independent Assessments: https://jac-admin-develop--pr2109-bugfix-2108-table-ov-ds494l0q.web.app/exercise/ONqUjAzhDS7rjh6dFJLB/tasks/independent-assessments#notrequested

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://github.com/jac-uk/admin/assets/79906532/e24999b1-dd30-4c72-9c87-18783c8dc763

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
